forms-flow-web/src/apiManager/endpoints/index.js

```diff
@@ -85,7 +85,9 @@ const API = {
   VALIDATE_TENANT: `${MT_ADMIN_BASE_URL}/${MT_ADMIN_BASE_URL_VERSION}/tenants/<tenant_id>/validate`,
   VALIDATE_FORM_NAME: `${WEB_BASE_URL}/form/validate`,
   EXPORT_FORM: `${WEB_BASE_URL}/form/<form_id>/export`,
-  GET_PROCESS_XML: `${WEB_BASE_URL}/process/key/<process_key>`
+  GET_PROCESS_XML: `${WEB_BASE_URL}/process/key/<process_key>`,
+  PUBLISH_BPMN: `${WEB_BASE_URL}/process/<process_id>/publish`,
+  UN_PUBLISH_BPMN: `${WEB_BASE_URL}/process/<process_id>/unpublish`
```

Comment:
is this endpoint common for BPMN & DPMN ? @abilpraju-aot

Reply:
For both dmn & bpmn same endpoint

Reply:
Please give a general name ( like PUBLISH_PROCESS , UNPUBLISH_PROCESS)